kegtab/src/main/java/org/kegbot/app/TapStatusFragment.java

```diff
@@ -256,6 +258,21 @@ public void onClick(View v) {
       description = tap.getDescription();
     }
 
+    // Find ABV and IBU values
+    final String abv = String.valueOf(keg.getBeverage().getAbvPercent());
+    if(mCore.getConfiguration().getAbvVisibleWhenZero()) {
+      abvText.setText("ABV: " + abv + "%");
+    } else{
+      abvText.setText(keg.getBeverage().getAbvPercent() == 0 ? "" : "ABV: " + abv + "%");
```

Comment:
This should probably just set the visibility to GONE. `abvText.setVisibility(View.GONE);`

Reply:
Agreed, fixing